### ISSUE TITLE
JDK21 serviceability/jvmti/stress/StackTrace/Suspended/GetStackTraceSuspendedStressTest.java Segmentation error vmState=0x00000000

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse-openj9/openj9#17782]
Failure link
------------

From [an internal build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk21_j9_extended.openjdk_aarch64_linux_Personal/6/consoleFull)(`ub20-aarch64-5`):
```
08:14:17  openjdk version "21-internal" 2023-09-19
08:14:17  OpenJDK Runtime Environment (build 21-internal-adhoc.jenkins.BuildJDK21aarch64linuxPersonal)
08:14:17  Eclipse OpenJ9 VM (build master-6bae0289b9e, JRE 21 Linux aarch64-64-Bit Compressed References 20230711_10 (JIT enabled, AOT enabled)
08:14:17  OpenJ9   - 6bae0289b9e
08:14:17  OMR      - bcc94e42952
08:14:17  JCL      - 73fae096864 based on jdk-21+30)
```
[Rerun in Grinder](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/parambuild/?SDK_RESOURCE=customized&TARGET=testList+TESTLIST=serviceability_jvmti_j9_0,serviceability_jvmti_j9_1,jdk_other_0,jdk_other_1,jdk_security_infra_0,jdk_security_infra_1&TEST_FLAG=&UPSTREAM_TEST_JOB_NAME=&DOCKER_REQUIRED=false&ACTIVE_NODE_TIMEOUT=&VENDOR_TEST_DIRS=functional&EXTRA_DOCKER_ARGS=&TKG_OWNER_BRANCH=adoptium%3Amaster&OPENJ9_SYSTEMTEST_OWNER_BRANCH=eclipse%3Amaster&PLATFORM=aarch64_linux&GENERATE_JOBS=true&KEEP_REPORTDIR=true&PERSONAL_BUILD=false&DOCKER_REGISTRY_DIR=&ADOPTOPENJDK_REPO=https%3A%2F%2Fgithub.com%2Fadoptium%2Faqa-tests.git&RERUN_ITERATIONS=0&SETUP_JCK_RUN=false&DOCKER_REGISTRY_URL_CREDENTIAL_ID=&LABEL=&EXTRA_OPTIONS=&BUILD_IDENTIFIER=fengj%40ca.ibm.com&CUSTOMIZED_SDK_URL=https%3A%2F%2Fna-public.artifactory.swg-devops.com%2Fartifactory%2Fsys-rt-generic-local%2Fhyc-runtimes-jenkins.swg-devops.com%2FBuild_JDK21_aarch64_linux_Personal%2F10%2FOpenJ9-JDK21-aarch64_linux-20230711-073423.tar.gz+https%3A%2F%2Fna-public.artifactory.swg-devops.com%2Fartifactory%2Fsys-rt-generic-local%2Fhyc-runtimes-jenkins.swg-devops.com%2FBuild_JDK21_aarch64_linux_Personal%2F10%2Ftest-images.tar.gz&JENKINS_KEY=Jenkins+4096+key&ADOPTOPENJDK_BRANCH=master&LIGHT_WEIGHT_CHECKOUT=true&USE_JRE=false&ARTIFACTORY_SERVER=na-public.artifactory.swg-devops&KEEP_WORKSPACE=false&USER_CREDENTIALS_ID=83181e25-eea4-4f55-8b3e-e79615733226&JDK_VERSION=21&DOCKER_REGISTRY_URL=&ITERATIONS=1&VENDOR_TEST_REPOS=git%40github.ibm.com%3Aruntimes%2Ftest.git&JDK_REPO=git%40github.com%3Aibmruntimes%2Fopenj9-openjdk-jdk21.git&JCK_GIT_BRANCH=main&OPENJ9_BRANCH=master&OPENJ9_SHA=6bae0289b9eb7748b84521bfd18d0cecf5c84785&JCK_GIT_REPO=&VENDOR_TEST_BRANCHES=master&OPENJ9_REPO=git%40github.com%3Aeclipse%2Fopenj9.git&UPSTREAM_JOB_NAME=&CLOUD_PROVIDER=&CUSTOM_TARGET=&VENDOR_TEST_SHAS=5ed35e82a0197057f8ce137ae4ab9e16bc9431d3&JDK_BRANCH=openj9&LABEL_ADDITION=ci.project.openj9&ARTIFACTORY_REPO=sys-rt-generic-local%2Fhyc-runtimes-jenkins.swg-devops.com&ARTIFACTORY_ROOT_DIR=&UPSTREAM_TEST_JOB_NUMBER=&DOCKERIMAGE_TAG=&JDK_IMPL=openj9&TEST_TIME=&SSH_AGENT_CREDENTIAL=83181e25-eea4-4f55-8b3e-e79615733226&AUTO_DETECT=true&SLACK_CHANNEL=%23rt-jenkins&DYNAMIC_COMPILE=false&RELATED_NODES=&ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH=adoptium%3Amaster&APPLICATION_OPTIONS=&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=7c1c2c28-650f-49e0-afd1-ca6b60479546&ARCHIVE_TEST_RESULTS=false&NUM_MACHINES=&OPENJDK_SHA=&TRSS_URL=http%3A%2F%2Ftrss1.fyre.ibm.com&USE_TESTENV_PROPERTIES=false&BUILD_LIST=openjdk&ADDITIONAL_ARTIFACTS_REQUIRED=&UPSTREAM_JOB_NUMBER=&STF_OWNER_BRANCH=adoptium%3Amaster&TIME_LIMIT=20&JVM_OPTIONS=&PARALLEL=None) - Change TARGET to run only the failed test targets.

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------
```
08:17:36  variation: Mode150
08:17:36  JVM_OPTIONS:  -XX:+UseCompressedOops 

08:19:16  TEST: serviceability/jvmti/stress/StackTrace/Suspended/GetStackTraceSuspendedStressTest.java

08:19:16  STDERR:
08:19:16  Unhandled exception
08:19:16  Type=Segmentation error vmState=0x00000000
08:19:16  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
08:19:16  Handler1=0000FFFFB4AF0004 Handler2=0000FFFFB4A56AF0 InaccessibleAddress=0000000000000AD0
08:19:16  R0=0000000000000000 R1=0000000000000000 R2=0000000000000000 R3=0000000000000001
08:19:16  R4=0000FFFFB001C208 R5=000000000000017F R6=0000FFFFB42CC221 R7=0000000069746D76
08:19:16  R8=0000FFFFB530F130 R9=000000000125DE8E R10=0000000000000018 R11=0024099DA3000000
08:19:16  R12=000048C175FF7120 R13=0000FFFF8D119430 R14=0000000000000000 R15=0000000000000050
08:19:16  R16=0000000000000001 R17=0000FFFFB525E5F8 R18=0000000000000000 R19=0000000000000000
08:19:16  R20=0000FFFF8D1194F8 R21=0000FFFFB42F1000 R22=0000FFFF1801ED20 R23=0000FFFFB002C270
08:19:16  R24=0000FFFFB00BBE38 R25=0000000000000000 R26=0000000000000000 R27=0000FFFF94060A68
08:19:16  R28=0000FFFF1801ED20 R29=0000FFFF8D119430 R30=0000FFFFB4290A90 R31=0000FFFF8D119430
08:19:16  PC=0000FFFFB4290AA0 SP=0000FFFF8D119430 PSTATE=0000000040000000
08:19:16  V0 00582bd6180118cf (f: 402725056.000000, d: 5.378278e-307)
08:19:16  V1 b55d4b46ad5c5389 (f: 2908509184.000000, d: -1.223376e-51)
08:19:16  V2 b55db4b918593319 (f: 408498976.000000, d: -1.240578e-51)
08:19:16  V3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V4 746e6f432f6d762f (f: 795702848.000000, d: 6.972913e+252)
08:19:16  V5 00000000e0108f28 (f: 3759181568.000000, d: 1.857282e-314)
08:19:16  V6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V7 00000000ffff8d28 (f: 4294937856.000000, d: 2.121981e-314)
08:19:16  V8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V16 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V17 0020000000100000 (f: 1048576.000000, d: 4.450148e-308)
08:19:16  V18 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V19 3f9eb851eb851eb8 (f: 3951369984.000000, d: 3.000000e-02)
08:19:16  V20 3fb1eb851eb851ec (f: 515396064.000000, d: 7.000000e-02)
08:19:16  V21 0000000000000800 (f: 2048.000000, d: 1.011846e-320)
08:19:16  V22 3fc999999999999a (f: 2576980480.000000, d: 2.000000e-01)
08:19:16  V23 3fa999999999999a (f: 2576980480.000000, d: 5.000000e-02)
08:19:16  V24 3fd6666666666666 (f: 1717986944.000000, d: 3.500000e-01)
08:19:16  V25 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V26 3fb999999999999a (f: 2576980480.000000, d: 1.000000e-01)
08:19:16  V27 000000000000000a (f: 10.000000, d: 4.940656e-323)
08:19:16  V28 0000000000000800 (f: 2048.000000, d: 1.011846e-320)
08:19:16  V29 0000000000000300 (f: 768.000000, d: 3.794424e-321)
08:19:16  V30 0000000000000000 (f: 0.000000, d: 0.000000e+00)
08:19:16  V31 0000000040400000 (f: 1077936128.000000, d: 5.325712e-315)
08:19:16  Module=/home/jenkins/workspace/Test_openjdk21_j9_extended.openjdk_aarch64_linux_Personal/openjdkbinary/j2sdk-image/lib/default/libj9jvmti29.so
08:19:16  Module_base_address=0000FFFFB4282000
08:19:16  Target=2_90_20230711_10 (Linux 5.4.0-153-generic)
08:19:16  CPU=aarch64 (8 logical CPUs) (0x1efe9d000 RAM)
08:19:16  ----------- Stack Backtrace -----------
08:19:16  jvmtiGetCarrierThread+0x130 (0x0000FFFFB4290AA0 [libj9jvmti29.so+0xeaa0])
08:19:16  _ZL18get_carrier_threadP9_jvmtiEnvP7JNIEnv_P8_jobject+0x40 (0x0000FFFF9405D320 [libGetStackTraceSuspendedStress.so+0x2320])
08:19:16  _ZL9agentProcP9_jvmtiEnvP7JNIEnv_Pv+0x19c (0x0000FFFF9405F9DC [libGetStackTraceSuspendedStress.so+0x49dc])
08:19:16  agent_thread_wrapper+0x3c (0x0000FFFF9405D70C [libGetStackTraceSuspendedStress.so+0x270c])
08:19:16  wrappedAgentThreadStart+0xb0 (0x0000FFFFB42ADDC0 [libj9jvmti29.so+0x2bdc0])
08:19:16  omrsig_protect+0x21c (0x0000FFFFB4A5777C [libj9prt29.so+0x2877c])
08:19:16  agentThreadStart+0x64 (0x0000FFFFB42ADB24 [libj9jvmti29.so+0x2bb24])
08:19:16  thread_wrapper+0xcc (0x0000FFFFB4A053BC [libj9thr29.so+0x73bc])
08:19:16  start_thread+0x184 (0x0000FFFFB525A624 [libpthread.so.0+0x7624])
08:19:16   (0x0000FFFFB519D49C [libc.so.6+0xd149c])
08:19:16  ---------------------------------------
08:19:16  JVMDUMP039I Processing dump event "gpf", detail "" at 2023/07/11 08:19:06 - please wait.

08:19:16  TEST RESULT: Failed. Unexpected exit from test [exit code: 255]
08:19:16  --------------------------------------------------
08:20:28  Test results: passed: 117; failed: 1
08:20:43  Report written to /home/jenkins/workspace/Test_openjdk21_j9_extended.openjdk_aarch64_linux_Personal/jvmtest/openjdk/report/html/report.html
08:20:43  Results written to /home/jenkins/workspace/Test_openjdk21_j9_extended.openjdk_aarch64_linux_Personal/aqa-tests/TKG/output_16890777958286/serviceability_jvmti_j9_0/work
08:20:43  Error: Some tests failed or other problems occurred.
08:20:43  -----------------------------------
08:20:43  serviceability_jvmti_j9_0_FAILED
```

[50x internal grinder](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/33922/) - 1 failure (in [grinder](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder_iteration_4/1887/) on ub20-aarch64-1)

**Comment by pshipton** [comment]
@babsingh

**Comment by babsingh** [comment]
Fixed by #17802.
